### PR TITLE
add network module to voltron

### PR DESCRIPTION
**Pull request by xiaoguo0742**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

File: driver/js/flutter/core/src/bridge/android/bridge_impl.cc [hunk 1]
```diff
@@ -265,3 +271,12 @@ void BridgeImpl::BindDomManager(int64_t runtime_id,
   runtime->GetScope()->SetDomManager(dom_manager);
   dom_manager->SetDelegateTaskRunner(runtime->GetScope()->GetTaskRunner());
 }
+
+std::shared_ptr<Scope> BridgeImpl::GetScope(int64_t runtime_id) {
+  std::shared_ptr<Runtime> runtime = Runtime::Find(static_cast<int32_t>(runtime_id));
```
Comment (churchill-zhang) [inline driver/js/flutter/core/src/bridge/android/bridge_impl.cc:276]: 这里的静态转换需要加宏检查，另外我听龙泉说他要改下整个runtime逻辑，内部都是int32_t，估计你要和他对下

File: driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc [hunk 2]
```diff
@@ -62,6 +63,10 @@ EXTERN_C void CreateInstanceFFI(int32_t engine_id, int32_t root_id, double width
     if (runtime && dom_manager) {
       auto runtime_id = runtime->GetRuntimeId();
       BridgeImpl::BindDomManager(runtime_id, dom_manager);
+#if ENABLE_INSPECTOR
+      auto scope = BridgeImpl::GetScope(runtime_id);
```
Comment (churchill-zhang) [inline driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc:67]: 我看前面设计，GetScope可能返回nullptr，这里不需要判空吗

File: driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc [hunk 3]
```diff
@@ -160,16 +166,78 @@ EXTERN_C void CallFunctionFFI(int32_t engine_id, const char16_t* action, const c
...
+  auto bridge_manager = BridgeManager::Find(engine_id);
+  if (bridge_manager) {
+    auto runtime = std::static_pointer_cast<FFIJSBridgeRuntime>(bridge_manager->GetRuntime().lock());
+    if (runtime) {
+      // change char16_t* to std::string
+      std::string request_string = hippy::base::StringViewUtils::ToU8StdStr(unicode_string_view(request_id));
+      std::string content_string = hippy::base::StringViewUtils::ToU8StdStr(unicode_string_view(request_content));
+
+      auto scope = BridgeImpl::GetScope(runtime->GetRuntimeId());
+      auto notification_center = scope->GetDevtoolsDataSource()->GetNotificationCenter();
```
Comment (churchill-zhang) [inline driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc:193]: 同上 判空

File: driver/js/flutter/core/src/bridge/android/bridge_impl.cc (hunk 1 above)
Comment (xiaoguo0742) [reply driver/js/flutter/core/src/bridge/android/bridge_impl.cc:276]: 已跟lucio对了，已修复

File: driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc (hunk 3 above)
Comment (xiaoguo0742) [reply driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc:193]: 已修复

File: driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc (hunk 2 above)
Comment (xiaoguo0742) [reply driver/js/flutter/core/src/ffi/bridge_ffi_impl.cc:67]: 已修复

**Review by JHGOGOGO — APPROVED** [review]

**Review by medns — APPROVED** [review]
LGTM

**Pull request merged**